sdk/digitaltwins/Azure.DigitalTwins.Core/README.md

```diff
@@ -6,26 +6,38 @@ This library provides access to the Azure Digital Twins service for managing twi
 
 ## Getting started
 
-### Authenticate the Client
+The complete Microsoft Azure SDK can be downloaded from the [Microsoft Azure Downloads][[microsoft_sdk_download]] Page and ships with support for building deployment packages, integrating with tooling, rich command line tooling, and more.
 
-### Install the package
+For the best development experience, developers should use the official Microsoft NuGet packages for libraries. NuGet packages are regularly updated with new functionality and hotfixes.
+
+## Prerequisites
+
+- Microsoft Azure Subscription: To call Microsoft Azure services, you need to create an [Azure subscription][azure_sub] 
+- Azure Digital Twin instance: In order to use the Azure Digital Twin SDK, you need to first create a Digital Twin instance using one of many options:
+  - Using [Azure portal][azure_portal] 
+  - Using [Azure Management APIs][azure_rest_api]
+  - Using [Azure CLI][azure_cli]: You will need to install azure cli and the [Azure IoT extension][iot_cli_extension] for Azure CLI. Refer to [IoT CLI Documentation][iot_cli_doc] for more information on how to create and interact with your Digital Twin instance. 
+
+## Install the package
 
 Install the Azure Digital Twins client library for .NET with [NuGet][nuget]:
 
 ```PowerShell
 Install-Package Azure.DigitalTwins.Core
 ```
-### Prerequisites
 
-* An [Azure subscription][azure_sub].
+## Authenticate the Client
+In order to interact with the Azure Digital Twin service, you will need to create an instance of a [ClientSecretCredential][client_secret_credential] and pass it to the constructor of your [DigitalTwinClient][digital_twin_client].
```

Comment:
Not just this type. Any type that derives from TokenCredential. You should include a link to the web docs for the options.

https://docs.microsoft.com/en-us/dotnet/api/azure.core.tokencredential?view=azure-dotnet